src/integration-test/java/org/web3j/quorum/ResponseTest.java

```diff
@@ -41,4 +56,157 @@ public void testPrivatePayload() {
         PrivatePayload privatePayload = deserialiseResponse(PrivatePayload.class);
         assertThat(privatePayload.getPrivatePayload(), is("0x"));
     }
+
```

Comment:
if you choose to split the `requestsTests` class, I would suggest to follow the same pattern here.

Reply:
agreed - this is done